contrib/mpi-proxy-split/record-replay.h

```diff
@@ -59,7 +59,7 @@
 // Returns true if we are currently replaying the MPI calls from the saved MPI
 // calls log; false, otherwise. Normally, this would be true while restoring
 // the MPI state at restart time. All other times, this would return false.
-#define LOGGING() \
+#define MPI_LOGGING() \
```

Comment:
Could you add a comment before macro stating that how `LOGGING` is conflicting with MPI's logging? And, what are some side effects of keeping the logging macro as `LOGGING`?

Reply:
Sure, will do.
Currently if we wanted to enable `JTRACE`, we have to `./configure --enable-logging`, and it won't compile since makefile passed `-DLOGGING` when compiling, and it's conflict with the definition here.